### ISSUE TITLE
Questions about ‘scale’ in cross entropy loss

### ISSUE THREAD

**Issue by Staten-Wang** [ShiqiYu/OpenGait#184]
A scale is multiplied in the input of cross-entropy loss. What is the meaning of this and what is the basis for it?
